sematic/ui/packages/main/src/pipelines/PipelineBar.tsx

```diff
@@ -188,7 +188,7 @@ export default function PipelineBar() {
           setSnackMessage({
             message: "New run available.",
             actionName: "view",
-            autoHide: false,
+            autoHide: true,
```

Comment:
because the old snack bar code was buggy. It was making this message autohide even if this was `false`. Now that I have fixed the code, we want this to continue auto-hiding (I believe that was the intended behavior, despite what this code says).